outputs.tf

```diff
@@ -63,3 +58,35 @@ output "additional_cidr_blocks_to_association_ids" {
     i.cidr_block => i.id
   }
 }
+
+output "additional_ipv6_cidr_blocks" {
+  description = "A list of the additional IPv4 CIDR blocks associated with the VPC"
+  value = [
+    for i in aws_vpc_ipv6_cidr_block_association.default :
+    i.ipv6_cidr_block
+  ]
+}
+
+output "additional_ipv6_cidr_blocks_to_association_ids" {
+  description = "A map of the additional IPv4 CIDR blocks to VPC CIDR association IDs"
+  value = {
+    for i in aws_vpc_ipv6_cidr_block_association.default :
+    i.ipv6_cidr_block => i.id
+  }
```

Comment:
```suggestion
  value = locals.enabled ? {
    for i in aws_vpc_ipv6_cidr_block_association.default :
    i.ipv6_cidr_block => i.id
  } : null
```

Reply:
It's not necessary to null the outputs when the module is disabled. Most of this module's outputs are `string` and are `""` when disabled. For `list` and `map` it is acceptable that the outputs be empty lists and maps, and you can argue it is even preferable. At least for this module it is consistent with the older outputs.